.github/workflows/test.yml

```diff
@@ -67,6 +67,8 @@ jobs:
       run: awk '/```python/{flag=1;next}/```/{flag=0}flag' docs/quickstart.md > quickstart.py &&  PYTHONPATH=. python quickstart.py
     - name: Run Pytest
       run: python -m pytest -n=auto test/ -k "not (test_efficientnet and models/test_train.py)"
+    - name: Run ONNX
+      run: CPU=1 python -m pytest -n=auto test/external/external_test_onnx_backend.py --tb=no --disable-warnings
```

Comment:
Why are these flags needed?

Reply:
I added CPU=1 here just to trigger test exclusion in `external_test_onnx_backend.py`
for example:
```
# Don't know how to treat special TensorProto like TensorProto.FLOAT8E4M3FN
if getenv("CPU") or getenv("TORCH"):
  backend_test.exclude('test_dequantizelinear_axis_cpu')
  backend_test.exclude('test_dequantizelinear_cpu')
```
Not sure if this is the best way to do this :/

I deleted --disable-warnings
--tb=no is needed since the outputted info on failed tests is unnecessarily verbose